functions.php

```diff
@@ -724,11 +725,15 @@ function export_graphs(&$export, $export_path) {
 				}
 			}
 
+            export_debug('There are ' . sizeof($graphs) . ' graphs not in hosts to export for all trees');
+
 			$hosts = db_fetch_cell_prepared('SELECT GROUP_CONCAT(DISTINCT host_id)
 				FROM graph_tree_items
 				WHERE graph_tree_id IN(?)',
 				array(implode(', ', $ntree)));
 
+            export_debug('There are ' . sizeof(explode(',',$hosts)) . ' hosts to export for all trees');
```

Comment:
This code may incorrectly indicate that there is a host to be exported:

```php
php > echo sizeof(explode(',',false));
1
```

Reply:
Yes, I was aware of this, but having some indicator was better than nothing at all for debugging.  I didn't find enough importance to attempt to solve this issue.

Reply:
In a datacenter, that can lead to confusion where they think something should have appeared bu hasn't especially when they look at their rsync or ftp logs and find nothing happened.  It makes it appear as if something is broken, when in truth the display is simply wrong. Best to check the value is false or empty before trying to explode it.